setup.py

```diff
@@ -27,5 +28,5 @@
     ],
     keywords=['mozilla', 'basket'],
     test_suite="basket.tests.TestBasketClient",
-    tests_require=['mock==1.0.1', 'unittest2==0.5.1'],
+    tests_require=['mock==1.0.1'],
```

Comment:
We should also get mock up to version 2.0.0 so that it should be fully compatible with `unittest.mock` in Python 3.3+, and then optionally install it since it's not required for Py3.